packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -130,16 +133,17 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({ title, patientUuid, sh
     dateOfOrder: formatDate(new Date(order.dateActivated)),
     orderType: capitalize(order.orderType?.display ?? '--'),
     order: order.display,
-    // order: (
-    //   <div>{order.display.length > 20 ? <OrderDisplayTooltip orderDisplay={order.display} /> : order.display}</div>
-    // ),
     priority: (
       <div style={{ background: orderPriorityToColor(order.urgency), textAlign: 'center', borderRadius: '1rem' }}>
         {capitalize(order.urgency)}
       </div>
     ),
     orderedBy: order.orderer?.display,
-    status: capitalize(order.fulfillerStatus ?? '--'),
+    status: (
+      <Tag type={orderStatusColor(order.fulfillerStatus)} className={styles.singleLineText}>
```

Comment:
We should also have a tooltip in here, to show the whole text when the text is long and is not shown fully in the table.

Reply:
These are predefined so there's no need for the tooltip here

Reply:
I am actually concerned about the class assigned to it. The CSS will trim the text, let's say the user has a zoomed in view.